src/Fabulous.XamarinForms/Attributes.fs

```diff
@@ -63,6 +63,15 @@ module SmallScalars =
             let expands = (encoded &&& 0x00000000FFFFFFFFUL) = 1UL
 
             LayoutOptions(alignment, expands)
+            
+    // should this be in Fabulous?        
+    module Float32 =
+        let inline encode (v: float32) : uint64 =
+            v |> float |> BitConverter.DoubleToInt64Bits |> uint64
+
+        let inline decode (encoded: uint64) : float32 =
+            encoded |> int64 |> BitConverter.Int64BitsToDouble |> float32
+        
```

Comment:
I think you're right. This needs to be moved to the core repo so other flavors can use it as well.

Reply:
So I should raise a PR on the core repo first?